content/en/docs/tasks/tools/included/_index.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
👋 @tomkivlin .
I am wondering if a slightly less-tab approach would be better.
Since you have separated the content into different pages, you could use the section _index page
to list the different installs by OS instead of including all the files within the main tab component.
Each page should have better navigation and can include tabs on the page.
I don't recall the original structure of this installation page (there have been a few iterations).
What do you think?

Reply:
Hi @kbhawkey - just so I check my understanding, the idea is that we have a separate new page per OS with links from the main `install-kubectl` page? Perhaps there would still be a couple of "included" files for the common snippets (e.g. gcloud installation steps and validation steps).  e.g.

```
tools/_index
tools/install-kubectl-linux
tools/install-kubectl-macos
tools/install-kubectl-windows
tools/included/kubectl-installs-gcloud
tools/included/kubectl-validate
```

I like the idea and have seen it in other places where per-OS install instructions are relatively complex (e.g. [Docker Engine](https://docs.docker.com/engine/install/) with a page per-OS rather than a tab per-OS).  If I've understood correctly, I'm happy to make those changes.

Reply:
OK, I have made the changes (see [here](https://deploy-preview-26682--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/)). 

One thing I've noticed is that the autocompletion instructions are tabbed with bash options for Linux and Mac, so I should modify so that the Bash on Linux steps only appear on the Linux page, Bash on Mac only on Mac page, zsh on all three.

Reply:
Ok, I have made those last modifications.  @kbhawkey @nate-double-u @sftim could you let me know what you think please?

https://deploy-preview-26682--kubernetes-io-master-staging.netlify.app/docs/tasks/tools/